components/list/README.md

```diff
@@ -0,0 +1,71 @@
+<span style="color:red; font-size:24px; text-transform: uppercase;">**In Development**</span>
+
+# List Component
+
+The `d2l-list` is the container to create a styled list. It requires the use of `d2l-list-item` as it's children.
+
+## Usage
+
+Import
+```js
+import '@brightspace-ui/core/components/list/list.js';
+```
+Then add the `d2l-list` component to your html and use the `d2l-list-item` to define the `d2l-list`'s children.
+```html
+<d2l-list>
+	<d2l-list-item>Item 1</d2l-list-item>
+	<d2l-list-item>Item 2</d2l-list-item>
+	<d2l-list-item>Item 3</d2l-list-item>
+</d2l-list>
+```
+
+Output
+
+![Basic List](./screenshots/d2l-list-basic.png?raw=true)
+
+# d2l-list
+`d2l-list` can be modified to display the data in slightly different ways.
+## Signature
+```html
+<d2l-list [divider="[all|top|bottom|middle|none]"] [divider-extend]>
+	<d2l-list-item>...</d2l-list-item>
+	...
+	<d2l-list-item>...</d2l-list-item>
+</d2l-list>
+```
+## Attributes
+- **divider**: Describes which dividers will be shown for the list.
+  - *all* (default): Shows all the dividers including dividers on the top and bottom of the list.
+  - *top*: Shows all the dividers except for the one at the bottom of the list.
+  - *bottom*: Shows all the dividers except for the one at the top of the list.
+  - *middle*: Only show the dividers between list items. The top and bottom dividers are hidden.
+  - *none*: Show no dividers.
+- **divider-extend**: A boolean attributes that extends the length of the dividers so the space between the list item contents and the left and right edges of the list dividers.
+
+## Slot
+- **default**: Only allows `d2l-list-item` in the slot. Can take multiple `d2l-list-item`s.
+
+## Styles
+-**d2l-item-divider-hover**: Add this class to enable showing the dividers on hover. This functionality will be removed and was added for testing hover states.
+
+# d2l-list-item
+`d2l-list-item` defines the items for a `d2l-list`.
+## Signature
+```html
+<d2l-list-item>
+	[slot: illustration-outer]
+	[slot: illustration]
+	[slot: default]
+	[slot: actions]
+</d2l-list>
+```
+
+## Slots
+ - **default**: This is for the main content area. You can add styles to the elements in this slot.
+   - *d2l-list-item-text*: This will act as the primary text. Limit is the text to two lines and will auto wrap. It will turn gain all the hover attributes if required.
```

Comment:
Now that we can't use `@apply` this will put adding the correct text to the user. Obviously, this isn't a great solution if all the lists should have the same text. Discover list doesn't so maybe it should? Another pattern would be nice if we could still get the desired text result with hover effects.

Reply:
Another PR should take the time to find a better solution.